src/vs/workbench/contrib/terminal/browser/xterm/quickFixAddon.ts

```diff
@@ -344,26 +347,18 @@ export async function getQuickFixesForCommand(
 							}
 							case TerminalQuickFixType.Opener: {
 								const fix = quickFix as ITerminalQuickFixOpenerAction;
-								const label = localize('quickFix.opener', 'Open: {0}', fix.uri.toString());
+								if (!fix.uri) {
+									return;
+								}
+								const uriLabel = (fix.uri.scheme === Schemas.http || fix.uri.scheme === Schemas.https) ? encodeURI(fix.uri.toString(true)) : labelService.getUriLabel(fix.uri);
+								const label = localize('quickFix.opener', 'Open: {0}', uriLabel);
```

Comment:
The `: ` could get lost in localization. Probably only the word `Open` should be localized

Reply:
@roblourens that'll probably break rtl languages